requirements.txt

```diff
@@ -5,6 +5,7 @@ certifi==2022.6.15
 charset-normalizer==2.1.0
```

Comment:
again not sure why these are showing up in the diff. they're already in main so they shouldn't. i think you need to update your branch